xroad-catalog-collector/src/main/java/fi/vrk/xroad/catalog/collector/util/OrganizationUtil.java

```diff
@@ -50,6 +53,22 @@ private OrganizationUtil() {
 
     }
 
+    public static JSONObject getCompany(String url, String businessCode) {
+        final String fetchCompaniesUrl = new StringBuilder().append(url)
+                .append("/").append(businessCode).toString();
+        String ret = null;
+        try {
+            ret = getResponseBody(fetchCompaniesUrl);
+        } catch (KeyStoreException e) {
```

Comment:
Should these be logged and not just printStackTrace?
If one of the exceptions happen this function returns normally. Is that intended?

Reply:
I have now added logging and changed the code, so that it will return an empty json in case of any of these errors.

Reply:
I guess it depends on the case which is better, empty json or throwing an exception.